src/studying.md

```diff
@@ -11,9 +11,10 @@ Study in Anki is limited to the currently selected deck as well as any
 subdecks it contains.
 
 On the decks screen, your decks will be displayed in a list. There are
-two number columns, 'due' and 'new'. 'Due' is the count of waiting
-reviews and cards currently in learning. 'New' is the number of new
-cards that are ready to be learnt that day.
+three columns. 'Due' is the count of waiting
+reviews. 'New' is the number of new
+cards that are ready to be learnt that day. The third column shows
+the number of cards currently in learning.
```

Comment:
Is  this order correct?

Reply:
Corrected. The second column is unlabelled, perhaps label it as "Learn" could be helpful, specially for new users.